### PR TITLE
Fix attendance overview

### PR DESCRIPTION
**Pull request by delenamalan**

### CODE REVIEW

File: tests/unit/test_attendance.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def setUp(self):
             attendance='P',
             member=jabu,
             meeting=old_parliament_meeting,
-            created_at='2019-01-01')
+            created_at='2019-08-01')
```
Comment (jbothma) [inline tests/unit/test_attendance.py]: why did you change this? Was it to make the test fail to confirm the bug before fixing it?

Comment (delenamalan) [reply tests/unit/test_attendance.py]: Both. It's a valid edge case, is it not?
